tests/e2e/steps_partial_set_security.go

```diff
@@ -2372,45 +2158,47 @@ func stepsModifyChain() []Step {
 		},
 
 		// 5. modify the chain from Opt In to Top 100%
-		{
-			Action: SubmitConsumerModificationProposalAction{
-				Chain:         ChainID("provi"),
-				From:          ValidatorID("alice"),
-				Deposit:       10000001,
-				ConsumerChain: ChainID("consu"),
-				TopN:          100,
-			},
-			State: State{
-				ChainID("provi"): ChainState{
-					Proposals: &map[uint]Proposal{
-						6: ConsumerModificationProposal{
-							Deposit: 10000001,
-							Chain:   ChainID("consu"),
-							Status:  strconv.Itoa(int(gov.ProposalStatus_PROPOSAL_STATUS_VOTING_PERIOD)),
-						},
-					},
-				},
-			},
-		},
-		{
-			Action: VoteGovProposalAction{
-				Chain:      ChainID("provi"),
-				From:       []ValidatorID{ValidatorID("alice"), ValidatorID("bob"), ValidatorID("carol")},
-				Vote:       []string{"yes", "yes", "yes"},
-				PropNumber: 6,
-			},
-			State: State{
-				ChainID("provi"): ChainState{
-					Proposals: &map[uint]Proposal{
-						6: ConsumerModificationProposal{
-							Deposit: 10000001,
-							Chain:   ChainID("consu"),
-							Status:  strconv.Itoa(int(gov.ProposalStatus_PROPOSAL_STATUS_PASSED)),
-						},
-					},
-				},
-			},
-		},
+		// PERMISSIONLESS: Opt-In ---> TopN IS NOT SUPPORTED AT THE CURRENT STATE OF WORK !!!
```

Comment:
What's the problem here? Isn't Top N is supported from an E2E test perspective or from the core logic of permissionless?

Reply:
I assumed that we will not support this feature in the core logic of permissionless at it's first implementation.
Test were failing and I supposed that it was due to that.
I this is supported I'll add a TODO to make this test pass. Let me know